packages/cuba-react-core/src/app/Security.ts

```diff
@@ -1,39 +1,56 @@
-import {action, IObservableArray, observable, ObservableMap} from 'mobx';
-import {CubaApp, EntityAttrPermissionValue, PermissionInfo, RoleInfo} from '@cuba-platform/rest';
+import {action, computed, observable, ObservableMap} from 'mobx';
+import {CubaApp, EntityAttrPermissionValue, EffectivePermsInfo} from '@cuba-platform/rest';
 import {getAttributePermission} from '@cuba-platform/rest/dist-node/security';
 
 export class Security {
 
-  @observable permissions?: IObservableArray<PermissionInfo>;
   @observable attrPermissionCache: ObservableMap<string, EntityAttrPermissionValue> = new ObservableMap();
-  @observable roles?: IObservableArray<RoleInfo>;
+  @observable effectivePermissions?: EffectivePermsInfo;
+  @observable private restSupportEffectivePerms: boolean = true;
```

Comment:
I'm concerned about this default (*true*). Since `getAttributePermission` is synchronous and not reactive we return unfair value here

Reply:
It's private flag. If we are using false here - it will not works on situation, when effectivePermissions load in progress.